src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/BlobStorageReadTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.features;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.palantir.docker.compose.DockerComposeRule;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.BlobStorageRead;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.PDF;
+
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+import java.util.List;
+import java.util.function.Consumer;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BlobStorageReadTest {
+
+    @ClassRule
+    public static DockerComposeRule docker = DockerComposeRule.builder()
+        .file("src/integrationTest/resources/docker-compose.yml")
+        .build();
+
+    @Mock
+    private Consumer<List<PDF>> pdfsConsumer;
+
+    private CloudBlobClient cloudBlobClient;
+
+    @Before
+    public void setup() throws URISyntaxException, InvalidKeyException, StorageException {
+        CloudStorageAccount account = CloudStorageAccount.parse("UseDevelopmentStorage=true");
+        cloudBlobClient = account.createCloudBlobClient();
+    }
+
+    @Test
+    public void connects_without_exceptions() throws URISyntaxException, StorageException {
+        CloudBlobContainer abc = cloudBlobClient.getContainerReference("abc");
+        abc.createIfNotExists();
+
+        BlobStorageRead blobStorageRead = new BlobStorageRead(cloudBlobClient, pdfsConsumer);
+        blobStorageRead.readBlobs();
```

Comment:
No asserts ? May be put something that no exception occurred.

Reply:
extra points for javadoc comment ;)